server/graphql/v2/query/collection/UpdatesCollectionQuery.ts

```diff
@@ -11,7 +12,12 @@ const UpdatesCollectionQuery = {
     ...CollectionArgs,
     tag: {
       type: new GraphQLList(GraphQLString),
-      description: 'Only from accounts that have one of these tags',
+      description: 'Only return updates from accounts that have one of these tags',
+    },
+    type: {
```

Comment:
Considering if this rather should be called `accountType`. And the `tag` argument called `accountTag`. Brevity vs clarity?

Reply:
Maybe. Because from the context, we are on searching for updates and we may be implying that we're filtering by update type and update tag, which are plausible things.

Reply:
I would be good with that change.

Reply:
Renamed these arguments (without deprecating since this new query has only been live for a few days).